src/main/java/net/starlark/java/eval/DescriptorCaches.java

```diff
@@ -26,16 +26,34 @@
 import net.starlark.java.annot.StarlarkAnnotations;
 import net.starlark.java.annot.StarlarkMethod;
 
-/** Helper functions for StarlarkMethod-annotated fields and methods. */
-final class CallUtils {
+/** Caches for method descriptor and related objects. */
```

Comment:
Let's call this DescriptorCache, because each instance is only a single cache.

"""A DescriptorCache is a cache of MethodDescriptors associated with a particular StarlarkSemantics. Every StarlarkThread holds a DescriptorCache for its semantics. A DescriptorCache may be shared between threads, since they are maintained in a global map accessed by {@link #forSemantics} to avoid creating them unnecessarily."""